platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -177,35 +187,82 @@ object Capture {
             )
         }
 
+        /**
+         * Initializes the Capture SDK in a dedicated background thread with the specified API key, providers, and configuration.
+         *
+         * Will emit a completion of [io.bitdrift.capture.CaptureResult] when is startAsync is done.
+         *
+         * Calling other SDK methods has no effect unless the logger has been initialized.
+         * Subsequent calls to this function will have no effect.
+         *
+         * @param apiKey The API key provided by bitdrift. This is required.
+         * @param sessionStrategy session strategy for the management of session id.
+         * @param completion The callback that is called when the start async operation is complete. Called on the main
+         *                   thread.
+         * @param configuration A configuration that is used to set up Capture features.
+         * @param fieldProviders list of extra field providers to apply to all logs.
+         * @param dateProvider optional date provider used to override how the current timestamp is computed.
+         * @param apiUrl The base URL of Capture API. Depend on its default value unless specifically
+         *               instructed otherwise during discussions with bitdrift. Defaults to bitdrift's hosted
+         *               Compose API base URL.
+         * @param context an optional context reference. You should provide the context if called from
+         * a [android.content.ContentProvider]
+         */
         @Synchronized
         @JvmStatic
         @JvmOverloads
-        internal fun start(
+        fun startAsync(
             apiKey: String,
             sessionStrategy: SessionStrategy,
+            completion: ((CaptureResult<Unit>) -> Unit),
```

Comment:
per kotlin conventions we should make this a trailing lambda https://kotlinlang.org/docs/lambdas.html#passing-trailing-lambdas